src/test/java/httpClientTests/HttpClientWidgetTests.java

```diff
@@ -0,0 +1,57 @@
+package httpClientTests;
+
+import httpclient.controllers.WidgetRequest;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.apache.hc.core5.http.HttpStatus;
+import org.json.JSONObject;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static httpclient.helpers.JSONHelper.getJSONValueByKey;
+import static httpclient.helpers.ResponseHandler.getIdFromResponse;
+import static httpclient.helpers.ResponseHandler.getResponseAsJSONObject;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class HttpClientWidgetTests extends HttpClientBaseTest {
+    private final String WIDGET_JSON_FILE_NAME = "newWidget";
+    private final int DASHBOARD_ID = 46;
+    private WidgetRequest widgetRequest = new WidgetRequest(HttpClientBaseTest.client);
+    private int newWidgetId;
+    private CloseableHttpResponse response;
+
+    @BeforeEach
+    protected void testDataPreparation() {
+        response = widgetRequest.createWidget(WIDGET_JSON_FILE_NAME);
+        newWidgetId = getIdFromResponse(response);
+    }
+
+    @Test
+    void newWidgetCreated() {
```

Comment:
please, adhere to the Java naming convetion

Reply:
done